tests/Unit/Models/SubscriptionTest.php

```
@@ -320,4 +343,50 @@ public function hasEvaluationBoardCoercesFalsy(): void
+    }
+
+    #[Test]
+    public function isAccessExpiredReturnsFalseBeforePendingCancellationDate(): void
+    {
+        $subscription = [
+            'status' => 'active',
+            'cancel_at_period_end' => 1,
+            'expires_at' => '2026-05-31 23:59:59',
+        ];
+
+        $result = Subscription::isAccessExpired($subscription, new \DateTimeImmutable('2026-05-30 12:00:00'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\Subscription' in method 'isAccessExpiredReturnsFalseBeforePendingCancellationDate'. (undefined)

(StaticAccess)